client/keys/show.go

```diff
@@ -110,8 +107,7 @@ func runShowCmd(cmd *cobra.Command, args []string) (err error) {
 		return errors.New("cannot use --output with --address or --pubkey")
 	}
 
-	bechPrefix, _ := cmd.Flags().GetString(FlagBechPrefix)
-	bechKeyOut, err := getBechKeyOut(bechPrefix)
+	bechKeyOut, err := getHexKeyOut("")
```

Comment:
```suggestion
	hexKeyOut, err := getHexKeyOut("")
```

will need to replace elsewhere

Reply:
Done, also enabled the test again